content/api/commands/session.md

```diff
@@ -31,9 +31,9 @@ Enabling this flag does the following:
   [`Cypress.Cookies.preserveOnce()`](/api/cypress-api/cookies#Preserve-Once) and
   [`Cypress.Cookies.defaults()`](/api/cypress-api/cookies#Defaults) methods.
 
-Because the page is cleared before each test,
-[`cy.visit()`](/api/commands/visit) must be explicitly called in each test to
-visit a page in your application.
+Because the page is cleared at the beginning of each test,
+[`cy.mount()`](/api/commands/mount) or [`cy.visit()`](/api/commands/visit) must
```

Comment:
I believe most if not all other references list visit before mount. This doesn't have any deeper meaning other than mount came later and has been plugged into content in that order.

Reply:
I actually did this deliberately. I think we should actually reword all of our other examples to put mount before visit because if we don't want to appear like we're favoring one over the other, the most sensible way to list things would be alphabetically. But I'm happy to change this to visit followed by mount to be more consistent. We could always re-order them in a separate story if we wanted. @amirrustam, thoughts?

Reply:
I put it back in b7f6e7298130cdfef7d3529f0f26a1dd7c2cf2d4 and we can revisit later if necessary.